litellm/main.py

```diff
@@ -859,6 +862,11 @@ def completion(
             raise ValueError(
                 f"Unable to map your input to a model. Check your input - {args}"
             )
+        
+        if user_id != None:
+            # Updates the cost to the user_id
+            project.update_cost(completion_obj=response, user=user_id)
```

Comment:
this would not work for streaming as that's directly returned - but i believe Logging.success_callback already receives both streaming + this resutl 

@WilliamEspegren thoughts?

Reply:
Yeah, missed that. Great that you noticed it 👍. Will try to do it with success_callback